README.md

```diff
@@ -103,6 +103,76 @@ security:
 
 ```
 
+## Usage
+
+### Publishable
+
+To set an entity component as publishable, use the following annotation:
+
+```php
+use Silverback\ApiComponentBundle\Annotation as Silverback;
+use Silverback\ApiComponentBundle\Entity\Utility\PublishableTrait;
+
+/**
+ * @Silverback\Publishable
+ */
+class Foo
+{
+    use PublishableTrait;
+```
+
+Default property is named `publishedAt` and association to original resource `publishedResource`. To customize those
+properties, update the annotation and your class:
+
+```php
+use Silverback\ApiComponentBundle\Annotation as Silverback;
+
+/**
+ * @Silverback\Publishable(fieldName="publicationDate", associationName="originalResource")
+ */
+class Foo
```

Comment:
because the trait defines `isPublished` and that will also be dependant on the field names chosen, perhaps it is good to have a `publishableInterface` which defines this as required... or perhaps it isn't really required? alternatively we could add the `isPublished` property to the serialized array programatically we think it'd be a useful field to return to the application/API user?

Reply:
I was thinking this method to be just a helper for the developer, but not required by the bundle. The publication date should be returned in the response for users with draft access only, so they will be able to modify it (or even see if it's a published or a draft resource.